UofS-Pulse-Binfo/rawphenotypes
# Issue #37   perform additional validation to new traits in stage 3

This update will perform validation to new traits detected in the spreadsheet file (stage 2: describe new trait). Only traits user marked/checked for saving will be affected by this requirement.

To test:
1. Prepare a data collection spreadsheet file by downloading from Instructions Page.
2. Add additional/new trait without indicating the (unit) part of a trait.
3. Upload file and continue to Stage 2 - Describe Traits. In this stage check the traits, to load
define trait form.
- Enter count, g, cm days for quantitative values, date for date in YYYY-MM-DD format and text for Alphanumeric data.
- Data that do not match the unit (eg. Yes or No value in a Count unit) will not be saved. 
4. After updating materialized view step in stage 3, additional step relating to new traits will indicate if new trait(s) failed validation or all new traits passed test.

NOTE: this requires Pull Request #38 Function to fetch support email - to provide the module with an email address to which it can email new traits that failed validation.

This required PR#38 will be used to provide an email address which will notify administrator/support team via email in case new traits failed validation in stage 3.
